src/components/GameView.js

```diff
@@ -22,7 +22,7 @@ export const GameView = (props) => {
     gameOver: false,
     gameOverType: "",
   });
-  const [player, setPlayer] = useState("");
+  const [player, setPlayer] = useState("white");
```

Comment:
Can you change this to `White` because we want the text to be like `White wins` not `white wins`

Reply:
This comment will be irrelevant when you make new state for orientation. But revert this back to `White` as well